webpack/assets/javascripts/react_app/components/CPUCoresInput/CPUCoresInput.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { WarningTriangleIcon } from '@patternfly/react-icons';
+import PropTypes from 'prop-types';
+import { HelpBlock } from 'patternfly-react';
+import NumericInput from '../common/forms/NumericInput';
+import './cpuCoresInput.scss';
+
+const CPUCoresInput = ({ label, maxValue }) => {
+  const [value, setValue] = useState(1);
+  return (
+    <div className="form-group">
+      <div className="col-md-10">
+        <NumericInput
+          value={value}
+          minValue={1}
+          onChange={v => setValue(v)}
+          label={label}
+        />
+        {value > maxValue && (
+          <HelpBlock>
+            <WarningTriangleIcon className="warning-icon" />
+            Specified value is higher than recommended maximum
+          </HelpBlock>
+        )}
+      </div>
+    </div>
+  );
+};
+
+CPUCoresInput.propTypes = {
+  label: PropTypes.string,
+  maxValue: PropTypes.number,
```

Comment:
It would be nice to add a description for each prop, see number 2-3: https://github.com/theforeman/foreman-js/tree/master/packages/stories#component-story-format-csf